src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const STATUS_CHECKED = 'STATUS_CHECKED';
```

Comment:
- [ ] Kindly be noticed that the value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`